test/opt/graphics_robust_access_test.cpp

```diff
@@ -1548,6 +1548,184 @@ OpFunctionEnd
                                                   true);
 }
 
+TEST_F(GraphicsRobustAccessTest, bug4166) {
```

Comment:
Could you give a name that describes what is being tested?  A name like "ReportChangedWithIndexChange", or anything better you can come up with.

Reply:
Done